xnb/objects/Texture2D.py

```diff
@@ -0,0 +1,89 @@
+
+from __future__ import annotations
+from typing import List
+from PIL import Image
+
+from ..constants import SurfaceFormat
+
+import logging
+import numpy
+import io
+
+class Texture2D:
+    def __init__(self) -> None:
+        self.width: int | None = None
+        self.height: int | None = None
+        self.surface_format: SurfaceFormat | None = None
+        self.textures: List[bytes] = []
+        self.image_data: bytes = bytes()
+        self.logger = logging.getLogger(__name__)
+
+    @property
+    def content(self) -> List[bytes]:
```

Comment:
This one feels a bit unnecessary here, so I would remove this, if it's unused, since the Texture object doesn't really inherit from an abstraction layer.